packages/js/data/src/crud/actions.ts

```diff
@@ -103,7 +121,22 @@ export const createDispatchActions = ( {
 		}
 	};
 
-	const updateItem = function* ( id: number, query: Partial< ItemQuery > ) {
+	const deleteItem = function* ( id: IdType, force = true ) {
```

Comment:
It feels a little weird that `force` is `true` by default, but it looks like it most cases, `force` is required to delete an item.  Soft deletion is actually the edge case, so if we wanted to soft delete a product: `deleteItem( 3, false )`.

Open to counterpoints on this.

Reply:
Same for me, I have the feeling that the `force` shouldn't be `true` by default, but I concur with you that it makes sense to leave it like that since almost every case uses it.